plugin-gradle/CHANGES.md

```diff
@@ -3,10 +3,12 @@
 We adhere to the [keepachangelog](https://keepachangelog.com/en/1.0.0/) format (starting after version `3.27.0`).
 
 ## [Unreleased]
+### Fixed
+* Make `BaseGroovyExtension` and `BaseKotlinExtension` public. ([#1912](https://github.com/diffplug/spotless/pull/1912))
 
 ## [6.23.0] - 2023-11-27
```

Comment:
also mention breaking change for previous release?

Reply:
That's a bug instead of intended behavior.

Reply:
ba92c77add9ee3f5e0484e6caa37486893629a34